common/genes/wc_genes_static_models.txt

```diff
@@ -1305,6 +1305,47 @@
 			}
 			girl = boy
 		}
+		static_lich = {
+			index = 87
+			male = {
+				1 = lich1gold
+				1 = lich1bronze
+				1 = lich1silver
+				1 = lich1pewter
+				1 = lich2gold
+				1 = lich2bronze
+				1 = lich2silver
+				1 = lich2pewter
+				1 = lich3gold
+				1 = lich3bronze
+				1 = lich3silver
+				1 = lich3pewter
+			}
+			female = male
+			boy = male
+			girl = male
+		}
+		static_lich_duke = {
+			index = 88
+			male = {
+				1 = lich4gold
+				1 = lich4bronze
+				1 = lich4silver
+				1 = lich4pewter
```

Comment:
You fix a problem you stated...
Place all golds near each other, then bronze, then silver, then pewter...
```
1 = lich1gold
1 = lich2gold
1 = lich3gold
1 = lich1bronze
etc
```

Reply:
Applied change in b39fff224466497201cf1050022d51a5fa8a5e4d. This makes it a bit more consistent, but doesn't fix the problem entirely. Bronze Lich models in particular seem to almost always become pewter-colored when they become Dukes.